django_cloud_deploy/tests/unit/skeleton/utils_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit test for django_cloud_deploy/skeleton/utils.py."""
+
+import shutil
+import tempfile
+import unittest
+
+from django.core import management
+
+from django_cloud_deploy.skeleton import utils
+
+
+class UtilTest(unittest.TestCase):
```

Comment:
Can you add a test where the file doesnt exist?

Reply:
Done.